components/dashboard/DashboardTransactionsRow.js

```diff
@@ -49,86 +53,120 @@ const TransactionImage = {
   )
 };
 
-const DashboardTransactionRow = props => (
-  <Row noGutters="true">
-    <Col xs="1" md="1">
-      <div className="dashboard-transaction-row-img">
-        {TransactionImage[props.txType]}
-      </div>
-    </Col>
-    <Col className="dashboard-transaction-row pl-0" xs="11" md="11">
+class DashboardTransactionRow extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      created: props.created === undefined ? new Date() : props.created,
+      createdStr: ""
+    };
+
+    this.tick = this.tick.bind(this);
+  }
+
+  componentDidMount() {
+    this.timer = setInterval(() => this.tick(), 1000);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.timer);
+  }
+
+  tick() {
```

Comment:
I think we can avoid code duplication with a separate component, which will handle the ticks.

Reply:
Yes, I did think of that initially. Anyway, have created a new Timer component to handle the ticks.